.codacy.yaml

```diff
@@ -1,2 +1,3 @@
 exclude_paths:
   - 'tests/**'
+  - 'sample_data/*.json'
```

Comment:
I did not test this, but if I understand the [syntax](https://support.codacy.com/hc/en-us/articles/115002130625-Codacy-Configuration-File) correctly, this would also cover the json files in the sub folders:
`- 'sample_data/**/*.json'`

Reply:
You are right, thinks. I did not pay attention to the sub-folders